source/install-atlas-cli.txt

```diff
@@ -389,10 +403,30 @@ Follow These Steps
 
          .. include:: /includes/steps-verify-atlas-cli.rst
 
+   .. tab:: Docker
+      :tabid: docker
+
+      To pull the latest {+atlas-cli+} Docker image, run the following
+      command:
+
+      .. code-block::
+
+         docker pull mongodb/atlas-cli
+
+      After you pull the {+atlas-cli+} Docker image, you can run
+      {+atlas-cli+} commands with Docker. 
+      
+      To run an {+atlas-cli+} command with Docker, add 
+      ``docker run --rm atlas-cli`` before the
```

Comment:
```suggestion
      ``docker run --rm mongodb/atlas`` before the
```

Reply:
The name of the image will be the exact name that was used to pull the image so mongodb/atlas:tag in our case

Reply:
I think it might be worth mentioning that if you do not specify the tag explicitly docker pull mongodb/atlas will default to mongodb/atlas:latest. For instance if users want to get version v1.9.1 they need to explicitly specify mongodb/atlas:v1.9.1

All available tags will be listed on this page (nothing on there yet): https://hub.docker.com/repository/docker/mongodb/atlas/tags?page=1&ordering=last_updated